Jenkinsfile

```diff
@@ -145,7 +145,8 @@ pipeline {
                                         --env "AWS_SECRET_ACCESS_KEY=test" \
                                         --env "MHS_DYNAMODB_ENDPOINT_URL=http://dynamodb:8000" \
                                         --env "FAKE_SPINE_ADDRESS=http://fakespine" \
-                                        --env "MHS_INBOUND_QUEUE_URL=http://rabbitmq:5672" \
+                                        --env "MHS_INBOUND_QUEUE_BROKERS=amqp://rabbitmq:5672" \
+                                        --env "MHS_INBOUND_QUEUE_NAME=inbound" \
```

Comment:
we have the same var at line nr 10:
MHS_INBOUND_QUEUE_NAME = "${ENVIRONMENT_ID}-inbound"

Reply:
ignore this comment